remodel/connection.py

```diff
@@ -11,16 +11,26 @@
 
 
 class Connection(object):
-    def __init__(self, db='test', host='localhost', port=28015, auth_key=''):
+    def __init__(self, db='test', host='localhost', port=28015, auth_key='',
+                 user='admin', password=''):
         self.db = db
         self.host = host
         self.port = port
         self.auth_key = auth_key
+        self.user = user
+        self.password = password
```

Comment:
That's right, if you really want to mess with it, you surely can. "Easier to ask for forgiveness than permission".

To me, the contract is pretty clear and the version compatibility is a big plus. Hence, I would go for the "readable" approach.